plugins/basic/python/tk_3dsmaxplus_basic/plugin_bootstrap.py

```diff
@@ -455,3 +462,64 @@ def _get_plugin_info():
 
     # return a dictionary with the required info
     return dict(plugin_id=plugin_id, base_configuration=base_configuration)
+
+
+def _menu_items_2025():
+    return {
+        3001: {
+            "name": "Log In to Flow Production Tracking...",
+            "action": _login_user,
+        },
+        3002: {
+            "name": "Learn about Flow Production Tracking...",
+            "action": _jump_to_website,
+        },
+        3003: {
+            "name": "Try PTR for Free...",
+            "action": _jump_to_signup,
+        },
+    }
+
+
+def _add_to_2025_menu():
+    def populate_menu(menuroot):
+        for code, menu_item in _menu_items_2025().items():
+            menuroot.additem(code, menu_item["name"])
+            menuroot.addseparator()
```

Comment:
One separator after each menu entry? This does not seem right

Reply:
I tried to keep the previous layout as much as I could. But since we should only rely on having a parent item for dynamically adding items, I think having separators doesn't make sense anymore. Removing them